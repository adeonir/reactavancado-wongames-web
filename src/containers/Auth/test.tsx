import { render, screen } from '@testing-library/react'

import { Auth } from '.'

describe('<Auth />', () => {
  it('should render the heading', () => {
    const {
      container: { firstChild },
    } = render(<Auth />)

    expect(screen.getByRole('heading', { name: /auth/i })).toBeInTheDocument()

    expect(firstChild).toMatchSnapshot()
  })
})
