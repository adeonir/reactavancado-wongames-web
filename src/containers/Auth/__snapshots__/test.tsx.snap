// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Auth /> should render the heading 1`] = `
<h1
  class=""
>
  Auth
</h1>
`;
